### PR TITLE
Feat show no of filters applied

### PR DESCRIPTION
**Pull request by shrilakshmishastry**
# Show number of filters applied
Currently, a number of filters applied is not shown above the filter symbol. After adding this feature number of filters applied will be shown.

# Follwing file got updated
- FilterPlay.jsx

#New Function added under above file
-  getAppliedFilter() : which takes applied filter as objects and returns the number of tags and filter applied 

# How Has This Been Tested?
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: src/common/search/FilterPlays.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Modal } from "common";
 import { getAllCreators, getAllLevels, getAllTags } from "meta/play-meta-util";
-import { useContext, useState } from "react";
+import { useContext, useState, useEffect } from "react";
```
Comment (atapas) [inline src/common/search/FilterPlays.jsx:3]: Where are you using useEffect?

File: src/common/search/FilterPlays.jsx [hunk 2]
```diff
@@ -80,19 +105,26 @@ const FilterPlays = () => {
...
+  const [noOfAppliedFilter, setnoOfAppliedFilter] = useState(0);
+
   useBackListener(({ action }) => {
     if (action === "POP") {
       console.log("POP");
       setModifiedFilterQuery({
         level: "",
         tags: [],
         creator: "",
+        labels: [],
```
Comment (atapas) [inline src/common/search/FilterPlays.jsx:117]: It is not multi-select then why level is an array?

File: src/common/search/FilterPlays.jsx [hunk 3]
```diff
@@ -80,19 +105,26 @@ const FilterPlays = () => {
...
+        creators: [],
       });
       setFilterQuery({
         level: "",
         tags: [],
         creator: "",
+        labels: [],
+        creators: [],
```
Comment (atapas) [inline src/common/search/FilterPlays.jsx:125]: Same question for creators

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/common/search/FilterPlays.jsx (hunk 3 above)
Comment (shrilakshmishastry) [reply src/common/search/FilterPlays.jsx:125]: It's been taken as an array in order to support for multi labels and creators' filter

File: src/common/search/FilterPlays.jsx (hunk 1 above)
Comment (shrilakshmishastry) [reply src/common/search/FilterPlays.jsx:3]: Will remove it. I added that earlier to have something which I handled in different way

File: src/common/search/FilterPlays.jsx [hunk 4]
```diff
@@ -140,7 +174,7 @@ const FilterPlays = () => {
         className="btn-filter"
         title="Filter Plays"
       >
-        {/*<div className="badge">8</div>*/}
+        <div className="badge">{noOfAppliedFilter}</div>
```
Comment (atapas) [inline src/common/search/FilterPlays.jsx:177]: Let's not show the badge if the count is 0.

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/common/search/FilterPlays.jsx [hunk 5]
```diff
@@ -139,8 +173,12 @@ const FilterPlays = () => {
         onClick={() => setShowModal(true)}
         className="btn-filter"
         title="Filter Plays"
-      >
-        {/*<div className="badge">8</div>*/}
+      >{
+          noOfAppliedFilter === 0 ?
```
Comment (atapas) [inline src/common/search/FilterPlays.jsx:177]: You can return null instead.

Comment (shrilakshmishastry) [reply src/common/search/FilterPlays.jsx:177]: done

**Review by atapas — APPROVED** [review]

**Pull request merged**